dvc/stage/utils.py

```diff
@@ -161,6 +161,7 @@ def compute_md5(stage):
     # Remove md5 and meta, these should not affect stage md5
     d.pop(stage.PARAM_MD5, None)
     d.pop(stage.PARAM_META, None)
+    d.pop(stage.PARAM_DESC, None)
```

Comment:
Forgot about it the last time :man_facepalming: :man_facepalming: :man_facepalming: